test/sql/cloud/test_cloud_cases.test

```diff
@@ -31,7 +47,7 @@ DELETE FROM test where i = 2;
 statement ok
 COMMIT;
 
-query I
+statement ok
```

Comment:
Was this an accident?

Reply:
Yeah probably... my formatter added the `group: [cloud]` on a later commit but I must have ended up with a wonky merge when rebasing. Will revert!